src/sparseml/pytorch/optim/modifier_quantization.py

```diff
@@ -138,11 +143,15 @@ def __init__(
         self._quantize_embeddings = quantize_embeddings
         self._reduce_range = reduce_range
         self._quantize_linear_activations = quantize_linear_activations
+        self._enable_int4_activations = enable_int4_activations
```

Comment:
Does it make sense not to hard code number of bits and the min/max range? Sth like "activation_bits" defining number of bits looks nicer to me for the recipe (also who knows in the future we want to do mix quantization)?

Reply:
If we can make it more general, I'm thinking we could also use this field to say that by default we don't quantize activation of a module. What we learned from quantizing BERT is that module/function activations should not be quantized (unless there're good reasons) to avoid rounding errors (this happens when the activation signals go into another module that need to be quantized, and then we have unnecessary double quant/dequant modules).

Reply:
great point on `activation_bits` @natuan. let's save the second suggestion for a future update since this controls quantization of all modules and we do need activation quantization to some level in our models still